lodestar/style.css

```diff
@@ -1219,7 +1219,6 @@ body:not(.logged-in) .lodestar-panel .jetpack-testimonial .entry-header {
 	color: #555;
 	font-size: 20px;
 	font-size: 1.25rem;
-	text-align: center;
```

Comment:
I think it would be best to target just list elements here, rather than removing the center alignment for all homepage content, otherwise, we lose the center alignment for all content. e.g. something like this:

```
.lodestar-intro .entry-content ul,
.lodestar-intro .entry-content ol {
	text-align: left;
}
```

Reply:
@mikachan I removed the `text-align: center` because it made the editor view reflect what is happening on the frontend. If we just target the list, it looks like this in the editor if a paragraph is left-aligned:

<img width="751" alt="Screenshot on 2022-08-08 at 10-02-16" src="https://user-images.githubusercontent.com/45246438/183436938-8359470e-5fc0-49e8-ba5d-9146977bb3dc.png">

but the frontend then shows:

<img width="1232" alt="Screenshot on 2022-08-08 at 10-01-57" src="https://user-images.githubusercontent.com/45246438/183437115-88b48c0c-9727-4ca1-addd-181abb5caf22.png">

If that is okay, I'm happy to change it.

Reply:
If a page is NOT used as the homepage all of the content is left-aligned (like it shows in the editor).  When used as a homepage the alignment changes to center ALL the content, which is quite different and seems like a bug.

As far as I can tell this fix brings a page's alignment into uh... alignment (pun intended) with what it shows everywhere else.  Thus it doesn't seem to be an issue with just the list items, but with all content.  Therefore the change that's currently made seems right to me.

Reply:
Ah OK, that sounds good to me and makes sense. I was thinking that un-centering the home page content would cause more issues, but aligning all the alignments 😁 sounds better.